.github/workflows/new-cvp-integration.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: "CV integration testing"
+
+# on: pull_request
+on: workflow_dispatch
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
```

Comment:
We should cancel in-progress on this one, as it could leave the cloudvision server in some weird state.